### PR TITLE
Add lists

### PR DESCRIPTION
**Pull request by tmampa**
# To-do List App

## - Users can
- Add tasks
- Change dates
- Delete tasks
- Edit tasks
- Add projects

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @tmampa, 

**This is a great project you have going on, well-done lad**. 
   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: src/modules/Task.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    return this.dueDate;
+  }
+
+  getDateFormatted() {
+    const day = this.dueDate.split('/')[0];
+    const month = this.dueDate.split('/')[1];
+    const year = this.dueDate.split('/')[2];
+    return `${month}/${day}/${year}`;
+  }
+}
```
Comment (mke2111) [inline src/modules/Task.js]: - [ ] Each task should at least contain a title, description, due date, and priority. Your task is missing the description and priority fields.

File: package.json [hunk 2]
```diff
@@ -1,26 +1,31 @@
...
+    "eslint-config-prettier": "^7.2.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-prettier": "^3.3.1",
+    "eslint-plugin-react": "^7.21.5",
+    "eslint-plugin-react-hooks": "^1.7.0",
+    "prettier": "^2.2.1",
+    "stylelint-config-standard": "^22.0.0",
+    "webpack": "^5.18.0",
+    "webpack-cli": "^4.8.0"
```
Comment (mke2111) [inline package.json]: - [ ] Add `webpack-dev-server` to your devDependencies.

File: package.json [hunk 3]
```diff
@@ -1,26 +1,31 @@
...
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --progress --watch",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix"
+  },
```
Comment (mke2111) [inline package.json]: - [ ] Add all the necessary run commands to your scripts.
```
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
```

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @tmampa, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: src/modules/Task.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  }
+
+  setDate(dueDate) {
+    this.dueDate = dueDate;
+  }
+
+  getDate() {
+    return this.dueDate;
+  }
+
```
Comment (mke2111) [inline src/modules/Task.js]: - [ ] As previously requested, add these fields to the task.
> Each task should at least contain a title, description, due date, and priority. Your task is missing the description and priority fields.

**Review by george-shammar — APPROVED** [review]
Hi @tmampa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**